components/raftstore/src/store/fsm/apply.rs

```diff
@@ -1508,18 +1489,41 @@ where
     }
 }
 
+#[cfg(any(test, feature = "testexport"))]
```

Comment:
I don't think it's a good idea. It means the test will not cover the production.

Reply:
Removed